### ISSUE TITLE
[NPC] [DESOLACE] Vendor-Tron 1000 & Super-Seller 680

### ISSUE THREAD

**Issue by Bonecramp** [dalaranwow/dalaran-wow#4696]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Though the caravan's are present, halted, and not where they ought to be, the two vendors who accompany these convoys are not present at any point along their expected patrol. 
The convoy is currently stationary at 66.7, 59.1. See screenshot.

**Expected behaviour**: If the convoys included the vendors, Super-Seller 960 and Vendor-Tron 1000, the cooking recipes [Lean Wolf Steak], [Jungle Stew], and [Heavy Kodo Stew] would be purchasable. If these Vendors would be added to their expected pauses along the patrol, Super-Seller 960 at 40.5, 79.3; and Vendor-Tron 1000 at 60.3, 38.1; acquiring these cooking recipes would be possible and available where expected. Thank you.

**Steps to reproduce the problem**:
![wowscrnshot_050617_175007](https://cloud.githubusercontent.com/assets/27168405/25776505/78761952-3285-11e7-9616-74dcbe39da5f.jpg)

1. Visit  40.5, 79.3 and 60.3, 38.1, the vendors are not present, nor are they present along the road looping around the zone between those two points (includes convoy present and stationary at 66.7, 59.1.
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Bonecramp** [comment]
Update:
Found [Recipe: Jungle Stew] on Nerrist in Strangle Thorn Vale 32.7, 29.2
Found [Recipe: Heavy Kodo Stew] on Kireena in the Desolace 51.1, 53.5
[Recipe: Lean Wolf Steak] is only available on Super-Seller 680 as mentioned above.

**Comment by TeridanDW** [comment]
Hello @Bonecramp, Thank you for the report. That's pretty interesting issue.

> By Woss (3,441 – 1·6·23) on 2009/02/04 (Patch 3.0.8)		
> I am replaying here,so my post is at the top of Comment List.
> 
> The Caravan Route is being described in this map.
> 
> When the Caravan is at the south edge of the route, Super-Seller 680 bot is spawning with some rare Cooking Recipies.
> 
> When the caravan is at the north edge of its route Vendor-Tron 1000 bot spawning with some other rare Cooking Recipies.
> Note 2: If you complete the quest that Cork Gizelton giving to you,caravan will accelarate its speed till it reach its next destination (north or south).
> Note 3: The caravan stays at its destination for 10Mins excactly.

I will need to investigate more.

**Comment by Synathy** [comment]
WoWHead links of the npcs
Gizelton Caravan Kodo : http://www.wowhead.com/npc=11564
Vendor-Tron 1000 : http://www.wowhead.com/npc=12245
Super-Seller 680 : http://www.wowhead.com/npc=12246
Cork Gizelton : http://www.wowhead.com/npc=11625
Rigger Gizelton : http://www.wowhead.com/npc=11626

**Comment by Rushor** [comment]
the event was perfectly scripted here https://github.com/dalaranwow/dalaran-wow/issues/443. the update for the behaviour will be apllied on the live realm in the future.

**Comment by nyrpha** [comment]
as of 29.11.2017 caravan npcs are still stationary and vendor-trons are missing, thus unable to buy the cooking recipes from them.

**Comment by Rahizm** [comment]
As of April 9, 2018, kodos are stuck in one place while the goblins are in different locations along their route. Neither vendor is spawned.

**Comment by Rahizm** [comment]
Update with new info. After a server reset, I went out and found Cork running south along his route. Cork stopped, yelled about needing a bodyguard, and gave quest before continuing his run. After 3 waves of kolkar, 2 kodos spawned and quest insta failed, but Cork kept running. After getting to south destination, Rigger and kodos spawned with Super Seller. 10-15 minutes later, Super Seller despawned, Cork took off running north route, and Rigger and kodos stayed in same spot. Rigger yelled about needing a bodyguard and offered quest. Cork ran a few hundred yards before stopping for 5-10 minutes and then continued running. Quest did not complete after getting through coven. Cork stopped running at 83.1,25.2 and got stuck there for over an hour before I left. It seems the problem lies in the pathing of the mobs.

**Comment by Jindra555** [comment]
28/05/2018, goblins are stuck on top of each other next to the Kolkar village, kodos near the spot where the road forks to Feralas. Neither moves and the vendor bot and super-seller aren't present.

**Comment by plantemarc** [comment]
![wowscrnshot_092018_165132](https://user-images.githubusercontent.com/42881489/45847225-5348ee80-bcf8-11e8-994b-7cc9b5efdb6d.jpg)

still bugged cant buy anything :(

**Comment by plantemarc** [comment]
Cork Gizelton and Rigger Gizelton and their both kodoo are stuck in the same place no sign of Vendor-Tron 1000 and 
![wowscrnshot_092018_165103](https://user-images.githubusercontent.com/42881489/45847426-00236b80-bcf9-11e8-9f0b-9732531b0f3d.jpg)
Super-Seller 680 :

**Comment by arestavo** [comment]
This is still a problem as of today. Rigger and Cork Gizelton are both stationary with their Kodos, and no Super-Seller 680 or Vendor-Tron 1000.
